src/view/com/posts/Feed.tsx

```diff
@@ -265,6 +266,12 @@ let Feed = ({
         )
       } else if (item === LOADING_ITEM) {
         return <PostFeedLoadingPlaceholder />
+      } else if (item.rootUri === 'fallback-marker-post') {
```

Comment:
Nit: could create a string constant for this to tie the two code locations together